File: app/src/main/java/com/cliff/conch/box/scene/ServiceManager.java
package com.cliff.conch.box.scene;

import android.os.IBinder;

import com.cliff.conch.box.reflect.RefClass;
import com.cliff.conch.box.reflect.RefStaticObject;

import java.util.Map;

public class ServiceManager {
    public static Class<?> TYPE = RefClass.load(ServiceManager.class, "android.os.ServiceManager");
    public static RefStaticObject<Map<String, IBinder>> sCache;
}

File: reflect-android/src/main/java/reflect/android/os/ServiceManager.kt
package reflect.android.os

import android.os.IBinder
import com.cliff.reflection.common.IReflect
import com.cliff.reflection.common.annotation.PStaticField
import com.cliff.reflection.common.annotation.ProxyClass

@ProxyClass("android.os.ServiceManager")
interface ServiceManager : IReflect {
    companion object

    @PStaticField
    val sCache: Map<String, IBinder>
}